### ISSUE TITLE
Add support for installing OpenFST bin tools when using CMake

### ISSUE THREAD

**Issue by asalarian** [kaldi-asr/kaldi#4263]
Currently the default make based installer, compiles and installs the OpenFST bin tools (e.g. `fstcompile`). However, the CMake based installer only compiles and installs the `fst` library. It would be great to add a switch to the CMake installer to allow installing these tools as well.

**Comment by danpovey** [comment]
But that's already done in `tools/`, which we'd run regardless of build
type.
So we'd need a switch there too, to avoid duplicate versions of the
binaries.

**Comment by asalarian** [comment]
> But that's already done in `tools/`, which we'd run regardless of build type. 
The way I understand it, the CMake build system does not use the scripts in `tool\`.

**Comment by danpovey** [comment]
Yes but if you actually want to use Kaldi you should probably run those
scripts anyway, as they install various things that are needed for the
example scripts to run.

**Comment by asalarian** [comment]
You are right as the CMake build system is still a work in progress so it might take some time before we have all features of Kaldi in it. However, in this case OpenFST already has support for building under CMake so the needed change on Kaldi side to support building and installing the bin tools of OpenFST in CMake is a simpler task.

**Comment by danpovey** [comment]
Yes, we might need to change the paths e.g. in tools/common_path.sh.
Unfortunately I dont have time to do this right now.  Maybe someone else
might.
